test/AccessTokenVerifier/AccessTokenVerifier.test.ts

```diff
@@ -34,7 +34,7 @@ describe("AccessTokenVerifier", function () {
     };
 
     this.value = {
-      expiry: Math.floor(new Date().getTime() / 1000) + 10,
+      expiry: Math.floor(new Date().getTime() / 1000) + 1000,
```

Comment:
Why this change?

Reply:
Tests fail because sometimes it takes more than 10 seconds to reach that part of the code, by which time the EAT has already expired xD

Reply:
Ooh